zipkin-server/src/test/java/zipkin2/server/internal/elasticsearch/ZipkinElasticsearchStorageConfigurationTest.java

```diff
@@ -302,8 +303,14 @@ public void doesntProvideBasicAuthInterceptor_whenBasicAuthUserNameandPasswordNo
     Client<HttpRequest, HttpResponse> decorated =
       factory.options.decoration().decorate(HttpRequest.class, HttpResponse.class, delegate);
 
+    HttpRequest req = HttpRequest.of(RequestHeaders.builder()
+      .method(HttpMethod.GET)
+      .scheme("http")
+      .authority("localhost")
+      .path("/")
+      .build()
+    );
     // TODO(anuraaga): This can be cleaner after https://github.com/line/armeria/issues/1883
```

Comment:
@anuraaga feel free to add a commit now or later as I couldn't quite grok where you were going on cleanup. In preparation, I had to add scheme and authority for this test to work again.